common/messages/ack.go

```diff
@@ -135,7 +135,7 @@ func (m *Ack) Validate(s interfaces.IState) int {
 	}
 
 	if delta > 30 {
-		return 0 // put this in the holding and validate it later
```

Comment:
Does this threshold still make sense w/ new Dependent holding?

Reply:
we need some limit. I have always thought that more than 11 minutes into the future was probably pointless but Paul wanted 30 so here we are.